docs/operations/troubleshooting.md

```diff
@@ -240,7 +240,7 @@ StorageClass expects it or that is has been deployed with a different name.
 ### Solution
 
 Deploy Ondat following the [installation procedures](
-/docs/introduction/quickstart). If you are using the manifests
+/docs/introduction/self-eval). If you are using the manifests
```

Comment:
https://docs.ondat.io/docs/introduction/self-eval is not a valid link - this needs to be https://docs.ondat.io/docs/self-eval/ instead.